configure.ac

```diff
@@ -26,6 +26,11 @@ LT_INIT([dlopen disable-fast-install win32-dll])
 #AC_LIBTOOL_DLOPEN
 AC_PROG_LIBTOOL
 
+#GSL is necessary for paraboloid polycapillary shapes, GSL in turn requires CBLAS and math libraries
+AC_CHECK_LIB([m],[cos])
+AC_CHECK_LIB([gslcblas],[cblas_dgemm])
+AC_CHECK_LIB([gsl],[gsl_blas_dgemm])
+
```

Comment:
Dit is niet de juiste manier vrees ik. Je moet het een beetje verder doen waar er gespeeld wordt met `PKG_CHECK_MODULES`. Voeg na de instructie waar er gezocht wordt naar `xraylib`, een gelijkaardig commando toe voor `gsl`:

````m4
PKG_CHECK_MODULES([gsl],[gsl])
````
Je zal zien dat deze lijn al een beetje verder staat, en opgeroepen wordt als easyRNG niet gevonden wordt. Verwijder het daar, en het zou in orde moeten zijn.